### ISSUE TITLE
Introduce attribute to disallow some POIs to be selected from "Edit POI" dialog

### ISSUE THREAD

**Issue by xmd5a2** [osmandapp/Osmand#1849]
1)Some POIs in poi_types.xml are "artificial" (created using entity_transform) and not intended to be added to OSM database. New xml attribute is needed to avoid that.
2)Tags with this attribute also should not be visible during editing existing POI in Advanced tab (tag list). Also additional + no_edit="true" tags. If such a tag is poi_type (primary tag), then do not open anything but raise popup poi_error_poi_not_found. See route_bus_ref https://dl.dropboxusercontent.com/u/4624786/osm/Screenshots/Screenshot_2015-11-10-17-34-07.png
3)Remove from "Advanced" - "Tag" list all tags with lang="true" (in poi_types.xml) except tag without ":_lang_" suffix.
```xml
	<poi_additional name="blind_description" tag="blind:description" type="text" lang="true"/>
```
https://dl.dropboxusercontent.com/u/4624786/osm/Screenshots/Screenshot_2015-11-11-14-24-30.png
There should be only blind:description.

**Comment by xmd5a2** [comment]
Tags with this attribute also should not be visible during editing existing POI in Advanced tab (tag list). If such a tag is poi_type (primary tag), then do not open anything but raise popup poi_error_poi_not_found.

**Comment by GaidamakUA** [comment]
Issue should be resolved by commit a80d791ea1ae533ed6b2ac96099c200f06906af5
